test.js

```diff
@@ -903,3 +904,15 @@ describe('Redis()', () => {
     });
   });
 });
+
+describe('Elastic()', () => {
+  test('Testing the name property which should be a string', (done) => {
+    const elastic = Elastic({ name: 'client-test' });
+
+    elastic.info(console.log);
+
+    elastic.end();
+
+    done();
```

Comment:
Unless you're inside a callback (which you might want to be with my change above), you don't need `done()` in sync paths.